pydruid/async_client.py

```diff
@@ -0,0 +1,157 @@
+#
```

Comment:
I changed the year in license header, from 2013 to 2016. Is that ok? If there is a completely new header somewhere, which is supposed to be used, could you point me to it?